ui/src/slices/experiments.ts

```diff
@@ -19,7 +19,10 @@ export const getStateofExperiments = createAsyncThunk(
   async () => (await api.experiments.state()).data
 )
 
-export const getNamespaces = createAsyncThunk('common/chaos-available-namespaces', async () => (await api.common.chaosAvailableNamespaces()).data)
+export const getNamespaces = createAsyncThunk(
+  'common/chaos-available-namespaces',
+  async () => (await api.common.chaosAvailableNamespaces()).data
+)
```

Comment:
It seems there is no change in this file.

Could we revert it?

Reply:
this should be format by tools in vscode automatically, do you think the update is better? @g1eny0ung

Reply:
Yep. All codes used by the front will be formatted automatically before committing. These changes can be safely ignored.

 And this change should be revert because the latest code is already formatted.

Reply:
Oh, my mistake! I looked at the `dashboard_token_test`  branch before. It's no problem to format this part.

Reply:
OK，@ STRRL PTAL again